Assets/Scripts/LoudObjects/LoudObject.cs

```diff
@@ -45,17 +47,15 @@ void Update()
         } 
     }
 
-    void NotifyChasingNurse()
+    private void OnCollisionEnter(Collision other)
     {
-        //TODO: Add a parameter to this function, layer mask
-        Collider[] hitColliders = Physics.OverlapSphere(transform.position, loudEffectRadius);
-        foreach (Collider collider in hitColliders)
-        {
-            // if collider is the chasing nurse 
-            // {
-            // call it's function 
-            // }
-        }
+        if (!hasBeenBumped || hasHit || other.gameObject.CompareTag("Player")) return;
+        
+        Vector3 hitPoint = other.GetContact(0).point;
+        Vector3 hitNormal = other.GetContact(0).normal;
+
+        gameObject.GetComponent<NoisePing>().SpawnNoisePing(hitPoint, hitNormal);

```

Comment:
It would also be nice I think to be able to define the radius on the throwable, and maybe be able to pass it into this function? I can see `NoisePing` as something we don't have to edit and rather just does the ping at whatever speed / duration the "user" (in this case the Throwable) decides on?

Stuff like floorOffset and stuff can probably just be variables we don't touch inside that class of course though.

Any thoughts?